### DISCUSSION TITLE
Using /actions/runs?event=push I am unable to get more than 1000 results

### DISCUSSION THREAD

**Discussion by AndersAppelt** [community/community#32403] [Apps, API and Webhooks]
When I call `/actions/runs?event=push&page=1&per_page=100` I get 100 results as expected. The response also states `"total_count": 1909`. Everything works fine up to and including `page=10&per_page=100` which means that I can get exactly 1000 results.

But when I call `/actions/runs?event=push&page=11&per_page=100` I suddenly get

````
{
  "total_count": 0,
  "workflow_runs": []
}
````

Is this by design or is there a bug here?

If I call the endpoint **without** the `event=push`, I'm able to page through all workflow runs without issue.

**Comment by Fivetran-VitalyKolesnikov** [comment]
I have the same problem with parameter "created"
